schema.sql

```diff
@@ -27,5 +27,43 @@ BEGIN;
 DELETE FROM animals;
 ROLLBACK;
 
+-- Create more tables
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/229288231-b117717a-2b02-4176-9485-25bb167f9adc.png)

**[OPTIONAL]**

- [ ] It would be very nice if you remove this description showed in the image above to make your readme clean and professional 😊